code/API_definitions/qod-api.yaml

```diff
@@ -348,31 +348,28 @@ components:
           description: Contains the reported event and applicable information
     
     EventReport:
-      type: object
-      required:
-        - eventType
-      properties:
-        eventType:
-          $ref: "#/components/schemas/SessionEventType"
+      oneOf:
```

Comment:
What's the idea behind usage of discriminator and oneOf with only one ref ?
It might make sense if we extract and common this endpoint for all Camara APIs. Is it the way we follow?

Reply:
I guess this makes sense as long as we foresee new values for eventType for in the future. We can leave this way, prepared for extension, or add the extension syntax when new schemas for other event reports appear